shared/constants/types/wallets.js

```diff
@@ -135,12 +137,25 @@ export type _Payment = {
   targetAccountID: ?string,
   targetType: string,
   time: ?number,
-  txID: string,
-  unread: boolean,
   worth: string,
   worthCurrency: string,
 }
 
+export type _PaymentResult = _PaymentCommon & {
+  // The section can be derived from statusDescription, so this can be
```

Comment:
What does this mean?

Reply:
We can't derive `section`. Core wants us to preserve `getPendingPaymentsLocal`/`getPaymentsLocal` -> "Pending"/"History" respectively, maybe we should change this comment to say that.